x-pack/test/functional/services/machine_learning/job_wizard_common.ts

```diff
@@ -21,6 +26,16 @@ export function MachineLearningJobWizardCommonProvider({ getService }: FtrProvid
     return !subSelector ? subj : `${subj} > ${subSelector}`;
   }
 
+  function additionalSettingsSectionSelector(subSelector?: string) {
+    const subj = 'mlJobWizardAdditionalSettingsSection';
+    return !subSelector ? subj : `${subj} > ${subSelector}`;
+  }
+
+  function newCustomUrlFormModal(subSelector?: string) {
```

Comment:
Similar to `additionalSettingsSectionSelector`:  this is not needed here.

Reply:
Check https://github.com/elastic/kibana/pull/52269/commits/dc359ba43bfcafb6702441408bb6456e6d380741